Sources/XCMetricsClient/Machine Name Reader/MachineNameReader.swift

```diff
@@ -37,6 +37,48 @@ class HashedMacOSMachineNameReader: MachineNameReader {
         if encrypted {
             return Host.current().localizedName?.md5()
         }
-        return Host.current().localizedName
+        return removeLocalizedSingleQuotesAndSpaces(Host.current().localizedName)
```

Comment:
It looks like we're missing the parameter label defined below.
```suggestion
        return removeLocalizedSingleQuotesAndSpaces(from: Host.current().localizedName)
```